sw.js

```diff
@@ -0,0 +1,38 @@
+---
+layout: null
+---
+
+importScripts( '{{ site.baseurl }}/cache-polyfill.js' );
+
+var filesToCache = [
+  // root
+  '{{ site.baseurl }}/',
+  '{{ site.baseurl }}/index.html',
+  // css
+  '{{ site.baseurl }}/assets/css/main.css',
+  '{{ site.baseurl }}/assets/css/normalize.css',
+  '{{ site.baseurl }}/assets/css/syntax.css',
+  // images
+  '{{ site.baseurl }}/assets/img/octocat.png',
+  // pages
+  {% for page in site.documents %}'{{ site.baseurl }}{{ page.url }}',{% endfor %}
```

Comment:
Same. I didn't even think to try `site.documents` before @contolini mentioned it.